backends/nxlog/render.go

```diff
@@ -389,17 +391,21 @@ func (nxc *NxConfig) Render() []byte {
 	return common.ConvertLineBreak(result.Bytes())
 }
 
-func (nxc *NxConfig) RenderToFile() error {
+func (nxc *NxConfig) RenderToString() string {
+	return string(nxc.RenderToString())
```

Comment:
this should be  `string(nxc.Render())`  otherwise it will trigger a nice stackoverflow ;-)